actionlist/app/components/action_item_body.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  actionContainer: {
+    backgroundColor: '#990000',
+    padding: 5,
+    borderRadius: 8,
+    width: 100,
+  },
+  bodyView: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+  routeContainer: {
+    backgroundColor: '#ffffff',
+    padding: 5,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: '#990000',
+    marginRight: 10,
+    width: 100,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  routeButton: {
+    fontSize: 15,
+    color: '#990000',
+  },
+});
+
+const routeLog = () => ({});
+
+const takeAction = () => ({});
+
+const ActionItemBody = ({ actionRequested,
```

Comment:
I think if there is a deeper level of nesting this would make reading it a little difficult. I think I will stick with what I have now. Let me know if I have to change it.